pkgs/applications/misc/dupeguru/default.nix

```
@@ -14,6 +14,12 @@ python3Packages.buildPythonApplication rec {
     fetchSubmodules = true;
   };
 
+  patches = [(fetchpatch {
+    name = "remove-m-from-so-var.patch";
+    url = "https://github.com/arsenetar/dupeguru/commit/bd0f53bcbe463c48fe141b73af13542da36d82ba.patch";
+    sha256 = "07iisz8kcr7v8lb21inzj1avlpfhh9k8wcivbd33w49cr3mmnr26";
+  })];
```

preferred style, looks better with many patches
```suggestion
  patches = [
    (fetchpatch {
      name = "remove-m-from-so-var.patch";
      url = "https://github.com/arsenetar/dupeguru/commit/bd0f53bcbe463c48fe141b73af13542da36d82ba.patch";
      sha256 = "07iisz8kcr7v8lb21inzj1avlpfhh9k8wcivbd33w49cr3mmnr26";
    })
  ];
```